src/cldfviz/templates/text/ExampleTable_detail.md

```diff
@@ -20,3 +23,18 @@
 {% endif %}
 >
 {% if ctx.cldf.translatedText != None %}> ‘{{ ctx.cldf.translatedText }}’{% endif %}
+
+{% else %}
+{% set pad_len = (example_id or ctx.id)|length+3 %}
```

Comment:
@xrotwang In addition to the four spaces needed for a code block, every line below the first is indented by the length of the example label + 2 spaces for the parentheses + 1 space for the space between the example label and the language name.